Runtime/Triangulator.cs

```diff
@@ -2660,29 +2658,17 @@ private static void Eigen(double2x2 matrix, out double2 eigval, out double2x2 ei
         private static double2x2 Kron(double2 a, double2 b) => math.double2x2(a * b[0], a * b[1]);
     }
 
-    internal readonly struct TranslationInt32 : ITransform<TranslationInt32, int, int2, long>
+    internal readonly struct TranslationInt : ITransform<TranslationInt, int, int2>
     {
-        public readonly TranslationInt32 Identity => new(int2.zero);
-        public readonly long AreaScalingFactor => 1;
-
+        public readonly TranslationInt Identity => new(int2.zero);
+        public readonly int AreaScalingFactor => 1;
         private readonly int2 translation;
-
-        public TranslationInt32(int2 translation) => this.translation = translation;
-        private static TranslationInt32 Translate(int2 offset) => new(offset);
-        public static TranslationInt32 operator *(TranslationInt32 lhs, TranslationInt32 rhs) => new(
-            lhs.translation + rhs.translation
-        );
-
-        public TranslationInt32 Inverse() => new(-translation);
+        public TranslationInt(int2 translation) => this.translation = translation;
+        public TranslationInt Inverse() => new(-translation);
         public int2 Transform(int2 point) => point + translation;
-        public long TransformArea(long area) => area * AreaScalingFactor;
+        public readonly TranslationInt CalculatePCATransformation(NativeArray<int2> positions) => throw new NotImplementedException();
```

Comment:
You are removing a useful error message here: `PCA is not implemented for integer coordinates`. This is very useful for users to diagnose why their code is throwing an exception.

Reply:
I'm preparing the documentation, this will be included there with information which features are supported for each type `T2`.